EIPS/eip-225.md

```diff
@@ -0,0 +1,474 @@
+ * **`nonce`**: Signer proposal regarding the account defined by the `beneficiary` field.
+   * Should be **`NONCE_DROP`** to propose deauthorizing `beneficiary` as a existing signer.
+   * Should be **`NONCE_AUTH`** to propose authorizing `beneficiary` as a new signer.
+   * **Must** be filled with zeroes on checkpoint (i.e. epoch transition) blocks.
+   * **Must** not take up any other value apart from the two above (for now).
+ * **`extraData`**: Combined field for signer vanity, checkpointing and signer signatures.
+   * First **`EXTRA_VANITY`** bytes (fixed) may contain arbitrary signer vanity data.
+   * Last **`EXTRA_SEAL`** bytes (fixed) is the signer's signature sealing the header.
+   * Checkpoint blocks **must** contain a list of signers (`N*20 bytes`) in between, **omitted** otherwise.
+   * The list of signers in checkpoint block extra-data sections **must** be sorted in ascending order.
+ * **`mixHash`**: Reserved for fork protection logic, similar to the extra-data during the DAO.
+   * **Must** be filled with zeroes during normal operation.
+ * **`ommersHash`**: **Must** be **`UNCLE_HASH`** as uncles are meaningless outside of PoW.
+ * **`timestamp`**: **Must** be at least the parent timestamp + **`BLOCK_PERIOD`**.
+ * **`difficulty`**: Contains the standalone score of the block to derive the quality of a chain.
+   * **Must** be **`DIFF_NOTURN`** if `BLOCK_NUMBER % SIGNER_COUNT != SIGNER_INDEX`
+   * **Must** be **`DIFF_INTURN`** if `BLOCK_NUMBER % SIGNER_COUNT == SIGNER_INDEX`
+
+### Authorizing a block
+
+To authorize a block for the network, the signer needs to sign the block's hash containing **everything except the signature itself**. The means that the hash contains every field of the header (`nonce` and `mixDigest` included), and also the  `extraData` with the exception of the 65 byte signature suffix. The fields are hashed in the order of their definition in the yellow paper.
+
+This hash is signed using the standard `secp256k1` curve, and the resulting 65 byte signature (`R`, `S`, `V`, where `V` is `0` or `1`) is embedded into the `extraData` as the trailing 65 byte suffix.
+
+To ensure malicious signers (loss of signing key) cannot wreck havoc in the network, each singer is allowed to sign **maximum one** out of **`SIGNER_LIMIT`** consecutive blocks. The order is not fixed, but in-turn signing weighs more (**`DIFF_INTURN`**) than out of turn one (**`DIFF_NOTURN`**).
+
+#### Authorization strategies
+
+As long as signers conform to the above specs, they can authorize and distribute blocks as they see fit. The following suggested strategy will however reduce network traffic and small forks, so it's a suggested feature:
+
+ * If a signer is allowed to sign a block (is on the authorized list and didn't sign recently).
+   * Calculate the optimal signing time of the next block (parent + **`BLOCK_PERIOD`**).
+   * If the signer is in-turn, wait for the exact time to arrive, sign and broadcast immediately.
```

Comment:
Can we explicitly define how in-turn signers and out-of-turn signers are determined?

Something like:
Given a dynamic list of signers, the in-turn signer for block at height h would be the signer at index i where i = h modulo length of signers.

There is sufficient data in the document to figure this out, however it may be helpful were it in its own section.